core/src/test/resources/org/everit/jsonvalidator/testschemas.json

```diff
@@ -543,5 +543,44 @@
         }
       }
     }
+  },
+  "defaultsTest": {
```

Comment:
Would you please also add a test with `"default": null` ? The tricky thing there is that the org.json library represents null values with `JSONObject#NULL` instead of a null reference, so maybe you will have to improve the `#deepToOrgJson()` method to cover it.